src/vs/workbench/contrib/webview/browser/webviewElement.ts

```diff
@@ -419,6 +419,7 @@ export class WebviewElement extends Disposable implements IWebview, WebviewFindD
 		// The extensionId and purpose in the URL are used for filtering in js-debug:
 		const params: { [key: string]: string } = {
 			id: this.id,
+			parentId: this._windowId!.toString(),
```

Comment:
You can read that out of `targetWindow.vscodeWindowId` and avoid the `!` assertion